### ISSUE TITLE
Intégrer les épisodes spéciaux

### ISSUE THREAD

**Issue by mkalam-alami** [supercinemarbre/supercinemarbre.github.io#23]
Dont le 47.5 spécial feux de cheminée

**Comment by 520-STB** [comment]
Ceux là sont en dehors du classement général.
Je pourrai ajouter un onglet par épisode special dans la Google  sheet ?

**Comment by mkalam-alami** [comment]
> Je pourrai ajouter un onglet par épisode special dans la Google sheet ?

Oui ça servirait merci, je pourrai faire en sorte d'intégrer ces films hors-classement.

**Comment by 520-STB** [comment]
J'ai commencé a reprendre les épisodes spéciaux, dans un nouvel onglet "Episodes Spéciaux":
https://docs.google.com/spreadsheets/d/1_h4Yh9xU72iqH3gZI6OquYG-jfBYPP4d1k-T9jwxEq8/edit#gid=1808501027

Pour l'instant il y a Super Mission Impossible Battle et Super Star Wars Battle. 
On peut aussi faire un onglet par episode si tu préfère.

**Comment by mkalam-alami** [comment]
C'est bien exploitable comme tu l'as mis en forme, merci.

Pour info ce sera un petit chantier pour moi pour bien supporter ces listes spéciales, donc t'étonnes pas si ça tarde à arriver sur Super Ciné Marbre !

**Comment by 520-STB** [comment]
@mkalam-alami Pour me faire la main, j'ai modifié le script d'import google sheet pour charger les épisodes spéciaux dans un csv à part [https://github.com/supercinemarbre/supercinemarbre.github.io/tree/feature/import-specialep-timestamps] (feature/import-specialep-timestamps)

**Comment by mkalam-alami** [comment]
Top, effectivement avec ce format pas grand-chose à coder :) Hésite pas à merge la branche, à moins que tu prévoies de faire la partie Vue dans la foulée ?

**Comment by 520-STB** [comment]
Oui, j'y réfléchi. Je pensais ajouter un menu "épisodes spéciaux" avec une présentation similaire à la page "épisodes" et en triant les films par leur classement.
Je me documente sur les bases de Vue pour l'instant, car je n'ai pas encore mis mon nez dedans.
